compiler/rustc_hir_typeck/messages.ftl

```diff
@@ -138,6 +139,11 @@ hir_typeck_option_result_asref = use `{$def_path}::as_ref` to convert `{$expecte
 hir_typeck_option_result_cloned = use `{$def_path}::cloned` to clone the value inside the `{$def_path}`
 hir_typeck_option_result_copied = use `{$def_path}::copied` to copy the value inside the `{$def_path}`
 
+hir_typeck_pass_to_variadic_function = can't pass `{$ty}` into variable arguments
```

Comment:
some options:
```suggestion
hir_typeck_pass_to_variadic_function = can't pass `{$ty}` into variadic arguments
```

```suggestion
hir_typeck_pass_to_variadic_function = can't pass `{$ty}` into `...`
```

Reply:
prefer either of these 👍

Reply:
In any case (esp. if we were to take the "`` `...` ``" one, we might also want to add a secondary highlight to the def site of the CVarArgs with a label to clear up any potential confusion.